### DISCUSSION TITLE
How to decrease Teleperiod-Time by "Berry" or "Scripting Language" to lower than 10 second

### DISCUSSION THREAD

**Discussion by HaraldNoack** [arendst/Tasmota#17675] [Q&A]
Hello,

I wrote a Berry-driver for a fast sensor and need MQTT-updates faster than 10 seconds.
How can I trigger a MQTT-Sensor-Message manually by "Berry" or "Scripting Language" not "Rules".
Best would be an update every second.

Thanks!
Best Regards
Harald

**Comment by barbudor** [comment]
Just directly publish what you want with `mqtt.publish()` (Berry)

**Reply by sfromis** [reply]
When you want that fast updates, it is no longer within the concept of sensors and telemetry. Anyway, you can issue a command (`Status 10` or `TelePeriod`) resulting in the same data being published, but there are reasons why 10 seconds was set as a minimum for regular stable operation. With Berry, you can also publish the data from `tasmota.read_sensors()`

**Reply by HaraldNoack** [reply]
Thanks but how do I write/format the commands?

tasmota.cmd("script Status 10")  
or
tasmota.cmd("script TelePeriod") 

Thanks!

**Reply by barbudor** [reply]
Why not simply `tasmota.cmd("Teleperiod")`  ?

**Reply by sfromis** [reply]
It is commands directly, you should not go via the script command too. Basically exactly the same commands you can do in the console.
